src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -112,7 +112,7 @@ public async Task UpdateFileStorageAsync()
 
                 if (!string.IsNullOrEmpty(sonarProjectKey))
                 {
-                    var allSuppressedIssues = serverIssuesStore.Get();
+                    var allSuppressedIssues = serverIssuesStore.Get().Where(x => x.IsResolved).ToArray();
```

Comment:
Still a bit unsure about linq syntax so if this is wrong lmk

Reply:
The syntax is fine.

You could skip the `ToArray` here, though; `allSuppressedIssues` is used in another LINQ expression immediately below which also has a `ToArray`, so we probably don't need it here as well.

An alternative approach would be to add the check into the LINQ expression below i.e.

```
       var allIssues = serverIssuesStore.Get();

       var settings = new RoslynSettings
                    {
                        SonarProjectKey = sonarProjectKey,
                        Suppressions = allIssues
                                            .Where(x => x.IsSuppressed)        <----- CHECK ADDED HERE
                                            .Select(x => IssueConverter.Convert(x))
                                            .Where(x => x.RoslynLanguage != RoslynLanguage.Unknown && !string.IsNullOrEmpty(x.RoslynRuleId))
                                            .ToArray(),
                    };

```

(I would probably go with the second approach and put all of the LINQ conditions in one place).